Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM java:8-jre
+FROM openjdk:8-jre
 MAINTAINER Gabe Conradi <gabe@tumblr.com>
```

Comment:
Is that alias accessible from outside the company? I thought there was a public maintainers mailing list for collins?